### ISSUE TITLE
PRINTING_MAILHOST_FIXED_ADDRESS handles the e-mail address as a list of chars

### ISSUE THREAD

**Issue by fredvd** [collective/Products.PrintingMailHost#4]
At least with some add'ons (Products.EasyNewsletter), the address specified as a fixed address is passed in as a string where PrintingMailhost considers it as a list. The MailHost object then start to send e-mails to every single letter of the address, which will fail.

Need to check if this also happens for for example the generic contact form and if it's a general bug or something also triggered by weirdness in the add'on.

**Comment by mauritsvanrees** [comment]
Works fine locally. I end up in `smtplib.py` of core Python 2.7.12, which takes care of it this way:

```
    if isinstance(to_addrs, basestring):
        to_addrs = [to_addrs]
```

I don't notice anything relevant in the release notes https://hg.python.org/cpython/raw-file/v2.7.12/Misc/NEWS

Looking in the Python sourcecode, this has been in there since 1998:
https://hg.python.org/cpython/annotate/3bdd0e790bc9/Lib/smtplib.py#l383

Locally I get a greylist error, which is irrelevant here, but the traceback shows the code that is in use around sending mail:

```
Traceback (most recent call last):
  File "/Users/maurits/shared-eggs/Products.CMFPlone-4.3.11-py2.7.egg/Products/CMFPlone/patches/sendmail.py", line 12, in _catch
    return func(*args, **kwargs)
  File "/Users/maurits/shared-eggs/zope.sendmail-3.7.5-py2.7.egg/zope/sendmail/mailer.py", line 77, in send
    connection.sendmail(fromaddr, toaddrs, message)
  File "/usr/local/pythons/parts/opt/lib/python2.7/smtplib.py", line 742, in sendmail
    raise SMTPRecipientsRefused(senderrs)
```

It could be that a different code path is taken.  This is when testing from the mail control panel form.

Anyway, it is easy enough to do a similar fix in our code.

**Comment by mauritsvanrees** [comment]
I have released 1.0 with a fix. :-)
